sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/ServiceCallAdapters.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk.impl
+
+import com.akkaserverless.javasdk
+import com.akkaserverless.scalasdk
+import com.akkaserverless.scalasdk.Metadata
+import com.akkaserverless.scalasdk.ServiceCall
+import com.akkaserverless.scalasdk.ServiceCallFactory
+import com.akkaserverless.scalasdk.ServiceCallRef
+import com.akkaserverless.scalasdk.SideEffect
+import com.google.protobuf
+import com.google.protobuf.Descriptors
+
+private[scalasdk] case class ScalaServiceCallFactoryAdapter(javaSdkServiceCallFactory: javasdk.ServiceCallFactory)
+    extends ServiceCallFactory {
+
+  override def lookup[T](serviceName: String, methodName: String, messageType: Class[T]): ServiceCallRef[T] =
+    ScalaServiceCallRefAdapter(javaSdkServiceCallFactory.lookup(serviceName, methodName, messageType))
+
+}
+private[scalasdk] case class JavaServiceCallAdapter(scalaSdkServiceCall: ServiceCall) extends javasdk.ServiceCall {
+  override def ref(): javasdk.ServiceCallRef[_] = JavaServiceCallRefAdapter(scalaSdkServiceCall.ref)
+  override def message(): protobuf.Any = scalaSdkServiceCall.message
```

Comment:
we probably want to use ScalaPB Any for the Scala SDK, but that can be a separate issue

Reply:
I will change it on another PR.